content/tutorials/react-setup.md

```diff
@@ -91,11 +91,9 @@ async componentDidMount() {
     nodeUri: "http://localhost:8545",
     aquariusUri: "http://localhost:5000",
     brizoUri: "http://localhost:8030",
+    brizoAddress: "0x00bd138abd70e2f00903268f3db08f2d25677c9e",
```

Comment:
This `brizoAddress` is different from the one in https://github.com/oceanprotocol/pleuston/pull/193/files

I'm not sure where to look up the `brizoAddress` for a particular network (a local Barge one in this case).

Reply:
This one is for spree network, new one for nile network is 0x376817c638d2a04f475a73af37f7b51a2862d567.
Looks like I left old one in Pleuston that is not corrent anymore.